packages/playground/_stories/main/RatingIndicator/RatingIndicator.stories.ts

```diff
@@ -23,42 +24,42 @@ export default {
     argTypes,
 } as Meta<RatingIndicator>;
 
-const Template: UI5StoryArgs<RatingIndicator, StoryArgsSlots> = (args) => html`<div></div>`;
+const Template: UI5StoryArgs<RatingIndicator, StoryArgsSlots> = (args) => html`
+<ui5-rating-indicator
+	value="${ifDefined(args.value)}"
+	max="${ifDefined(args.max)}"
+	?disabled="${ifDefined(args.disabled)}"
+	?readonly="${ifDefined(args.readonly)}"
+	accessible-нame="${ifDefined(args.accessibleName)}"
+></ui5-rating-indicator>`;
 
+export const BasicRatingIndicator = Template.bind({});
+BasicRatingIndicator.args = {
+	value: 3.7
+};
+BasicRatingIndicator.storyName = "Basic Rating Indicator";
```

Comment:
no need of the story name as the story variable name will result with the same

Reply:
Done